Changes any dependencies on PIL to use PIL.Image instead of Image

diff -Naur mcomix-1.00.orig/mime/comicthumb mcomix-1.00/mime/comicthumb
--- mcomix-1.00.orig/mime/comicthumb	2013-06-10 23:48:46.799633918 +0800
+++ mcomix-1.00/mime/comicthumb	2013-06-10 23:52:53.641991528 +0800
@@ -22,7 +22,7 @@
 import subprocess
 
 try:
-    import Image
+    import PIL.Image as Image
 except ImportError:
     print '! Could not import the Image module (PIL).'
     print __doc__
